platform-api/src/routers/platform.py

```
@@ -0,0 +1,138 @@
+
+@router.get("/health")
+async def aggregated_health(settings: WorkerSettings = Depends(get_settings)):
+    """Unauthenticated aggregated health check across all external services."""
+    checks = {}
+    
+    async def probe(name: str, coroutine):
+        try:
+            # Enforce 5-second timeout for external probes (AC3)
+            await asyncio.wait_for(coroutine, timeout=5.0)
+            checks[name] = "ok"
+        except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)